From c400e93b63866f692e603f3181cc66d1097c8eb3 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Fri, 17 Jul 2020 18:59:22 +0300
Subject: [PATCH 068/105] imx8: eeprom: Move to the imx8 platform independent
 folder

Move the eeprom code the imx8 platform independent folder.
The same code gets used by all CompuLab imx8 boards.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 board/compulab/iot-gate-imx8/Makefile                    | 2 ++
 board/compulab/mcm-imx8m-mini/Makefile                   | 2 ++
 board/compulab/plat/imx8/Makefile                        | 1 +
 board/compulab/plat/{imx8mm => imx8}/eeprom/Makefile     | 0
 board/compulab/plat/{imx8mm => imx8}/eeprom/eeprom.c     | 0
 board/compulab/plat/{imx8mm => imx8}/eeprom/eeprom.h     | 0
 board/compulab/plat/{imx8mm => imx8}/eeprom/eeprom_spl.c | 0
 board/compulab/plat/imx8mm/Makefile                      | 1 -
 board/compulab/ucm-imx8m-mini/Makefile                   | 2 ++
 9 files changed, 7 insertions(+), 1 deletion(-)
 create mode 100644 board/compulab/plat/imx8/Makefile
 rename board/compulab/plat/{imx8mm => imx8}/eeprom/Makefile (100%)
 rename board/compulab/plat/{imx8mm => imx8}/eeprom/eeprom.c (100%)
 rename board/compulab/plat/{imx8mm => imx8}/eeprom/eeprom.h (100%)
 rename board/compulab/plat/{imx8mm => imx8}/eeprom/eeprom_spl.c (100%)

diff --git a/board/compulab/iot-gate-imx8/Makefile b/board/compulab/iot-gate-imx8/Makefile
index af743b977b..e5239d1dbb 100644
--- a/board/compulab/iot-gate-imx8/Makefile
+++ b/board/compulab/iot-gate-imx8/Makefile
@@ -5,6 +5,7 @@
 #
 
 ccflags-y += -I$(srctree)/board/compulab/plat/imx8mm/
+ccflags-y += -I$(srctree)/board/compulab/plat/imx8/
 
 ifdef CONFIG_SPL_BUILD
 obj-y += spl.o
@@ -13,3 +14,4 @@ obj-y += iot-gate-imx8.o
 endif
 
 obj-y += ../plat/imx8mm/
+obj-y += ../plat/imx8/
diff --git a/board/compulab/mcm-imx8m-mini/Makefile b/board/compulab/mcm-imx8m-mini/Makefile
index 9ce2875d6e..2cadd46fb8 100644
--- a/board/compulab/mcm-imx8m-mini/Makefile
+++ b/board/compulab/mcm-imx8m-mini/Makefile
@@ -5,6 +5,7 @@
 #
 
 ccflags-y += -I$(srctree)/board/compulab/plat/imx8mm/
+ccflags-y += -I$(srctree)/board/compulab/plat/imx8/
 
 ifdef CONFIG_SPL_BUILD
 obj-y += spl.o
@@ -13,3 +14,4 @@ obj-y += mcm-imx8m-mini.o
 endif
 
 obj-y += ../plat/imx8mm/
+obj-y += ../plat/imx8/
diff --git a/board/compulab/plat/imx8/Makefile b/board/compulab/plat/imx8/Makefile
new file mode 100644
index 0000000000..9d3044cb9e
--- /dev/null
+++ b/board/compulab/plat/imx8/Makefile
@@ -0,0 +1 @@
+obj-y += eeprom/
diff --git a/board/compulab/plat/imx8mm/eeprom/Makefile b/board/compulab/plat/imx8/eeprom/Makefile
similarity index 100%
rename from board/compulab/plat/imx8mm/eeprom/Makefile
rename to board/compulab/plat/imx8/eeprom/Makefile
diff --git a/board/compulab/plat/imx8mm/eeprom/eeprom.c b/board/compulab/plat/imx8/eeprom/eeprom.c
similarity index 100%
rename from board/compulab/plat/imx8mm/eeprom/eeprom.c
rename to board/compulab/plat/imx8/eeprom/eeprom.c
diff --git a/board/compulab/plat/imx8mm/eeprom/eeprom.h b/board/compulab/plat/imx8/eeprom/eeprom.h
similarity index 100%
rename from board/compulab/plat/imx8mm/eeprom/eeprom.h
rename to board/compulab/plat/imx8/eeprom/eeprom.h
diff --git a/board/compulab/plat/imx8mm/eeprom/eeprom_spl.c b/board/compulab/plat/imx8/eeprom/eeprom_spl.c
similarity index 100%
rename from board/compulab/plat/imx8mm/eeprom/eeprom_spl.c
rename to board/compulab/plat/imx8/eeprom/eeprom_spl.c
diff --git a/board/compulab/plat/imx8mm/Makefile b/board/compulab/plat/imx8mm/Makefile
index 2f68e0523e..33b3fedef9 100644
--- a/board/compulab/plat/imx8mm/Makefile
+++ b/board/compulab/plat/imx8mm/Makefile
@@ -1,3 +1,2 @@
 obj-y += ddr/
-obj-y += eeprom/
 obj-y += spl/
diff --git a/board/compulab/ucm-imx8m-mini/Makefile b/board/compulab/ucm-imx8m-mini/Makefile
index dbde2116cd..870792441c 100644
--- a/board/compulab/ucm-imx8m-mini/Makefile
+++ b/board/compulab/ucm-imx8m-mini/Makefile
@@ -5,6 +5,7 @@
 #
 
 ccflags-y += -I$(srctree)/board/compulab/plat/imx8mm/
+ccflags-y += -I$(srctree)/board/compulab/plat/imx8/
 
 ifdef CONFIG_SPL_BUILD
 obj-y += spl.o
@@ -13,3 +14,4 @@ obj-y += ucm-imx8m-mini.o
 endif
 
 obj-y += ../plat/imx8mm/
+obj-y += ../plat/imx8/
-- 
2.11.0

